Pipfile

```diff
@@ -7,6 +7,19 @@ name = "pypi"
 falcon = "~= 2.0.0"
 gunicorn = "~= 19.9"
 jsonschema = "~= 3.0"
+numpy = "*"
```

Comment:
All packages should have their versions pinned.

Reply:
Is there a fats way to do it, or it needs to be done by hand? Also, I don't really see the point of pinning the version of external dependencies, since it isn't used in the snekbox code directly, and therefore can't break it. Plus, it will avoid us to manually update a package when a new version is out.

Reply:
> Is there a fast way to do it, or it needs to be done by hand?

Maybe `pip freeze` can help, otherwise by hand. 

> since it isn't used in the snekbox code directly, and therefore can't break it

Still would rather not have anything break even if it's one package.

> Plus, it will avoid us to manually update a package when a new version is out.

Fair point perhaps. However, I don't think this would happen often so I'm OK with manual updates. Also, either way, a rebuild will still need to be triggered.'

Let's see what anyone else has to say.

Reply:
Let's just pin'em.